### ISSUE TITLE
[nit] in spaces build logs, `hf_transfer` is replaced with `hf_********` as if it was a token

### ISSUE THREAD

**Issue by severo** [huggingface/huggingface_hub#2430]
```
Consider using `hf_********` for faster uploads. This solution comes with some limitations. See https://huggingface.co/docs/huggingface_hub/hf_******** for more details.
```

**Comment by Wauplin** [comment]
Yes that has been reported [on slack](https://huggingface.slack.com/archives/C048K60MPNF/p1719399065975289?thread_ts=1719398190.366449&cid=C048K60MPNF)  (internal). It is not `huggingface_hub`-related. The logs from Spaces are filtered to remove tokens which should not happen for `hf_transfer`. @christophe-rannou would you be able to have a look at it server-side?

**Comment by christophe-rannou** [comment]
It is indeed due to the way we filter logs in spaces backend. Fix about to be released.

**Comment by Wauplin** [comment]
hf_transfer not recommended anymore, let's close
